src/DataViews/include/DataViews/DataView.h

```diff
@@ -180,6 +183,29 @@ class DataView {
     return nullptr;
   }
 
+  ErrorMessageOr<void> ExportToCSVFile(const std::string& file_path);
```

Comment:
Nit: Change `CSV` to `Csv` and also remove "File" from the name to be consistent with other methods.